browser/debounce/debounce_browsertest.cc

```diff
@@ -182,18 +176,6 @@ class DebounceBrowserTest : public BaseLocalDataFilesBrowserTest {
   }
 
   bool InstallAdBlockForDebounce() {
```

Comment:
I'm happy we're not using `InstallExtension` here anymore, but this method name no longer makes sense